### DISCUSSION TITLE
Not render html

### DISCUSSION THREAD

**Discussion by darkdim** [larowlan/vite-plugin-twig-drupal#7] [General]
Hi
I am getting html rendering in my components. Any ideas for my case?
I am using this installation option https://storybook.js.org/tutorials/intro-to-storybook/react/en/get-started/

**Comment by larowlan** [comment]
Are you using the `framework: 'react'` option with `@storybook/react-vite` ?

I saw this the other day and went back to resolve it and it magically had resolved itself.

But I'm keen to get to the bottom of it
